float64[] command
string[] names
---
